CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### New features
 
+* [#6151](https://github.com/rubocop-hq/rubocop/issues/6151): Clarify Lint/UnneededSplatExpansion's message. ([@tejasbubane][])
```

Comment:
Since this update does not change the `Lint/UnneededSplatExpansion` cop's behavior, so changelog entry is unnecessary.

Reply:
I thought the message is user-facing, hence added. Will remove.